### ISSUE TITLE
Pacman Problems

### ISSUE THREAD

**Issue by FieryMewtwo** [arch-linux-gui/alg_repo#2]
> If this is the wrong place to ask this, please tell me where to ask this (**not** Discord, please.)

Every time I try to run a system upgrade with pacman, it gives me the option to import the below PGP keys:
![Screenshot at 2022-03-04 17-20-23](https://user-images.githubusercontent.com/77066742/156901653-46b42c35-c0c3-489f-8325-b3be0c91ffa0.png)

No matter what I choose, eventually, I get this error:
![Screenshot at 2022-03-04 17-21-56](https://user-images.githubusercontent.com/77066742/156901684-212ec7ae-5408-478d-b47e-403e06427ed8.png)

Any and all help is much appreciated!

**Comment by demonkilleralg** [comment]
Update archlinux-keyring

sudo pacman -S archlinux-keyring

**Comment by FieryMewtwo** [comment]
It worked! Thanks so much! :D

**Comment by lkelly93** [comment]
Having a similar issue with the installer, specifically the i3 installer. Let me know if this isn't the right spot for this. I'm using the GUI to install and it says 

-----
Package Manger Error
The Package Manager could not prepare updates. The command:
pacman -Sy
returned error code 1.
-----

So I tried running that command manually and I got the PGP error shown above. I tried doing running the provided command and it seems to fix the issue but the the installer still fails.
